Mergin/projects_manager.py

```diff
@@ -86,6 +87,22 @@ def create_project(self, project_name, project_dir, is_public, namespace):
         QApplication.setOverrideCursor(Qt.WaitCursor)
         try:
             self.mc.create_project(full_project_name, is_public)
+        except ClientError as e:
+            QApplication.restoreOverrideCursor()
+            msg = str(e)
+            # User friendly error messages
+            if e.http_error == 409:
+                msg = f'Project with the name "{project_name}" already exists in the workspace "{namespace}"\n Try renaming the project'
+            elif e.http_error == 422 and e.server_code == ErrorCode.ProjectsLimitHit.value:
+                msg = (
+                    "Maximum number of projects is reached. Please upgrade your subscription to create new projects\n"
+                    f"Maximum of projects: {e.server_response["projects_quota"]}"
+                )
+            elif e.http_error == 422 and e.server_code == ErrorCode.StorageLimitHit.value:
+                msg = f"{e.detail}]\nCurrent limit: {dlg.exception.server_response["storage_limit"]} Mb"
```

Comment:
Do we know if the `dlg.exception.server_reponse["storage_limit"]` always reports MBs?

Reply:
Added the function  `bytes_to_mb` in utils.py